webapp/src/octoClient.ts

```diff
@@ -11,16 +11,17 @@ import {Utils} from './utils'
 //
 class OctoClient {
     readonly serverUrl: string
-    token?: string
-    readonly readToken?: string
+    get token(): string {
+        return localStorage.getItem('sessionId') || ''
```

Comment:
I think the problem of this is, it is going to read to token every single time from the local storage, that is not the fastest thing ever. I would recommend to cache it in a private `_token` variable.

Reply:
I did this originally because I was worried about timing (when sessionIds change after init), so wanted to push the "source of truth" to the localStorage. There's an issue when sessionIds get stale (e.g. on refresh) in the OctoListener too.

Is localStorage perf a concern? I did a [quick search](https://stackoverflow.com/questions/8074218/speed-cost-of-localstorage) and it should be on the order of microseconds.

Might need to refactor to a central "sessionManager" later.